### ISSUE TITLE
states.schedule.present always removes "-enabled:True" every highstate run.

### ISSUE THREAD

**Issue by hal58th** [saltstack/salt#28052]
states.schedule.present always removes "-enabled:True" every highstate run. It doesn't seem to actually do anything in the long run, but it does always show the state as changed.

```
test_schedule:
  schedule.present:
    - function: cmd.run
    - job_args:
      - "echo hi"
    - minutes: 60
    - seconds: 45
```
```
          ID: test_schedule
    Function: schedule.present
      Result: True
     Comment: Modifying job test_schedule in schedule
     Started: 17:25:54.932424
    Duration: 192.795 ms
     Changes:
              ----------
              diff:
                  ---
                  +++
                  @@ -1,5 +1,4 @@
                   args:['echo hi']
                  -enabled:True
                   function:cmd.run
                   jid_include:True
                   maxrunning:1
```
```
salt-call schedule.list show_all=True
    schedule:
      __mine_interval:
        ...
        ...
      test_schedule:
        args:
        - echo hi
        enabled: true
        function: cmd.run
        jid_include: true
        maxrunning: 1
        minutes: 60
        name: test_schedule
        seconds: 45
```

```
Salt Version:
           Salt: 2015.8.1

Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: Not Installed
           Mako: 0.9.1
         PyYAML: 3.10
          PyZMQ: 14.4.0
         Python: 2.7.6 (default, Jun 22 2015, 17:58:13)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.4
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 1.5
          gitdb: 0.5.4
      gitpython: 0.3.2 RC1
          ioflo: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.3.0
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: 0.8.2
        timelib: Not Installed

System Versions:
           dist: Ubuntu 14.04 trusty
        machine: x86_64
        release: 3.13.0-24-generic
         system: Ubuntu 14.04 trusty
```

**Comment by jfindlay** [comment]
@hal58th, thanks for the report.  Ping @garethgreenaway.

**Comment by garethgreenaway** [comment]
Will take a look.

**Comment by cachedout** [comment]
Thanks, @garethgreenaway

**Comment by JaseFace** [comment]
Dupe of https://github.com/saltstack/salt/issues/27391

The fix was merged 16 days ago, but it appears to have missed 2015.8.1 which was 21 days ago.

**Comment by rallytime** [comment]
@JaseFace Oh, yeah, you're right. This is a duplicate. Your fix didn't quite make it in before we tagged the 2015.8.1 release, but it will be in 2015.8.2.

Since this is a duplicate and is already fixed on the 2015.8 branch, I am going to go ahead and close this.
